.chloggen/parallel-awscloudwatchlogsexporter.yaml

```diff
@@ -0,0 +1,22 @@
+# Use this changelog template to create an entry for release notes.
+# If your change doesn't affect end users, such as a test fix or a tooling change,
+# you should instead start your pull request title with [chore] or use the "Skip Changelog" label.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: 'enhancement'
```

Comment:
maybe breaking is ok? since the config is changed.

Reply:
That is correct, we are extending the configuration and keeping the previous behaviour unchanged. That is why I don't consider this a breaking change.